share/instruments/instruments.xml

```diff
@@ -1052,17 +1052,13 @@
                   <genre>earlymusic</genre>
             </Instrument>
             <Instrument id="recorder">
+                  <init>soprano-recorder</init>
                   <family>recorders</family>
```

Comment:
I guess we can drop this line too and inherit via that `<init>`, but I'm not sure and don#t know how to check